frontend/src/components/Quiz/QuestionReview.tsx

```diff
@@ -0,0 +1,70 @@
+import { useQuiz } from "../../contexts/quizContext";
+import {
+  VStack,
+  Heading,
+  Text,
+  Tag,
+  SimpleGrid,
+  Button,
+  Box,
+} from "@chakra-ui/react";
+import {
+  getPoints,
+  primaryButtonStyleProps,
+  questionStyleProps,
+} from "../../utils";
+import OptionReview from "./OptionReview";
+import { ArrowForwardIcon } from "@chakra-ui/icons";
+import { useNavigate } from "react-router";
+
+function QuestionReview() {
+  const {
+    state: { quizAttempt },
+  } = useQuiz();
+  const { questions, totalQuestions } = quizAttempt!;
+  const navigate = useNavigate();
+
+  return (
+    <>
+      <SimpleGrid gap={6} textAlign={"center"} mb={8}>
+        {questions.map((quizQuestion, index) => {
+          const { id, question, isAttempted, options, points, negativePoints } =
+            quizQuestion;
```

Comment:
Same Don't initialize or calculate anything in the map function of return statement. Keep the return statement only for rendering elements. This way your code looks more cleaner.

Reply:
Same reason as above